testsuite/pytests/sli2py_recording/test_multimeter_freeze_thaw.py

```diff
@@ -52,16 +57,30 @@ def simulate_freeze_thaw(num_neurons):
 
     nrns[0].frozen = True
     nest.Simulate(5.0)
+    nest.Simulate(5.0)
```

Comment:
Here and in a number of other places down to line 120 it seems as if some lines are doubled up in the new version.

Reply:
Duplicates from merge error removed in [4b020be](https://github.com/nest/nest-simulator/pull/2754/commits/4b020be7abd443702c60e241f987d8963979f8e5).